### PR TITLE
Update get materia guide, heroku setup, docker setup and git urls

### PR DESCRIPTION
**Pull request by iturgeon**
Now that Materia is open source, the process for getting materia has changed.  Updated the docs accordingly.

* Updated all github.com/ucfcdl links to ucfopen
* Cleaned up the Docker install instructions
* Added a page to index all the server setup options
* Added a Heroku setup page

### CODE REVIEW

File: _administrators/heroku.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+category: [server, server_setup]
+highlighter: no
+---
+# Test Drive Materia on Heroku
+
+Heroku is the fastest way to try Materia. If you don't know what Heroku is, they provide a service for running web servers in the cloud.  They manage all the hardware, allowing you to easily scale your service up and down to meet your needs.
+
+## Heroku Limitations
+
+Heroku will spontaneously destroy and create new application servers as needed. Files stored on disk after building the initial image will be lost in this process. Currently, Heroku Materia v4.0.0 is not able to install additional widgets unless you add them to the application config before deploying. See [Installing Widgets](#installing-widgets).
```
Comment (FrenjaminBanklin) [inline _administrators/heroku.md]: This block specified `Materia v4.0.0`. Would it make more sense to say `Materia v4.0.0 or higher`, just to specify the minimum Heroku-viable version? As is, another Materia release goes out the docs will be referring to an old version. Unless we're okay with tweaking this part of the docs with every release?

File: _administrators/heroku.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+Because of the disk storage hurdles, Heroku Materia installs will store uploaded media in the database.  This will quickly use up the [free database storage plan](https://elements.heroku.com/addons/cleardb) limit of 5mb. The database storage can be increased for a fee from the Heroku dashboard.  Or, for a more scalable option, media can be stored on Amazon S3.
+
+Heroku's implementation of Memcached is not compatible with Materia (via the underlying FuelPHP framework).  Therefore, Heroku Materia installs may be a _little bit_ slower.
+
+
+## Creating a New Heroku App
+
+Clicking the deploy to Heroku button below will begin building a server.  It'll install a few widgets and give you a public web address to begin playing with Materia.  The default build uses free assets, so it won't cost a thing. To get started, just click the deploy button below.
```
Comment (FrenjaminBanklin) [inline _administrators/heroku.md]: Nitpick: `the deploy to Heroku button` -> `the 'Deploy to Heroku' button`, just to match the image of the button below.

File: _administrators/ubuntu-apache-install-guide.md [hunk 3]
```diff
@@ -193,7 +193,7 @@ $ git config --global credential.helper 'cache --timeout=3600'
 
 Right now the Materia Github repositories are private, meaning you have to have credentials to access them.  Unfortunately this adds a few steps to the install.
 
-1. Make sure you have a Github user that can see [https://github.com/ucfcdl/Materia](https://github.com/ucfcdl/Materia)
+1. Make sure you have a Github user that can see [https://github.com/ucfopen/Materia](https://github.com/ucfopen/Materia)
 2. If you're using Github 2 factor authentiaction, [create an access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) to use as a password
```
Comment (FrenjaminBanklin) [inline _administrators/ubuntu-apache-install-guide.md]: Know it wasn't part of this PR, but may as well fix it while we can see it: 'authentiaction' -> 'authentication'.

File: get_materia.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+title: How to get Materia&trade;
+tagline: Get your own Materia&trade;
+class: admin
+highlighter: no
+---
+# Materia on Github
+
+Materia is open source, so the server and many of the widgets can be found on [Github](https://github.com/ucfopen/Materia)
```
Comment (FrenjaminBanklin) [inline get_materia.md]: Needs a period at the end.

File: _developers/style-guide.md [hunk 5]
```diff
@@ -94,4 +94,4 @@ PHP Code Sniffer is installed with Composer and included in the dev dependencies
 
 Basically, if you install Materia Server normally using Composer, then you should be all set.
 
-For detailed information see our [Github PHPCS rules for Materia and FuelPHP](https://github.com/ucfcdl/fuelphp-phpcs)
+For detailed information see our [Github PHPCS rules for Materia and FuelPHP](https://github.com/ucfopen/fuelphp-phpcs)
```
Comment (FrenjaminBanklin) [inline _developers/style-guide.md]: Needs a period at the end.

**Review by FrenjaminBanklin — CHANGES_REQUESTED** [review]

File: _administrators/heroku.md (hunk 1 above)
Comment (iturgeon) [reply _administrators/heroku.md]: I'll just leave the v4.0.0 bit out it made sense when I was writing (I think to get at exactly what you were suggesting).  But reading it without the version seems totally fine.

File: _administrators/heroku.md (hunk 2 above)
Comment (iturgeon) [reply _administrators/heroku.md]: Agree, fought with that idea myself.

File: _administrators/ubuntu-apache-install-guide.md (hunk 3 above)
Comment (iturgeon) [reply _administrators/ubuntu-apache-install-guide.md]: Looks like that whole section is irrelevant now that it's open source.

File: _administrators/server-docker-development.md [hunk 6]
```diff
@@ -2,9 +2,16 @@
...
 # Running a Development Docker Server
 
-Materia developers use Docker to quickly get a local copy of the server running.  For more information on developing using Docker, visit the [Developing Materia Server Guide](../develop/platform-developer-guide.html#docker-development-environment)
+Docker is the fastest way to get a local copy of Materia Server running. Materia has a built in [collection of scripts and configuration files](https://github.com/ucfopen/Materia/tree/master/docker) needed to orchestrate a local development Docker environment.
+
+Our developers use this setup to run tests, develop new features, and to qa widgets.  It has containers available for all optional hosting features in Materia.
+
+To get started, clone Materia, cd into the `docker` directory, and execute `./run_first.sh`.
+
+The [Materia Docker README](https://github.com/ucfopen/Materia/blob/master/docker/README.md) will get you up and running in less then 10 minutes.
```
Comment (FrenjaminBanklin) [inline _administrators/server-docker-development.md]: 'than'

File: _administrators/heroku.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+* Username: `~materia_system_only`
+* password: `<value of USER_SYSTEM_PASSWORD>`
+
+
+
+
+
+## Run Administration Commands
+
+To run any of the [administration commands]({{ site.baseurl }}/admin/administrative-commands.html) on your Heroku server, access the Run Console menu item from your Heroku Dashboard
```
Comment (FrenjaminBanklin) [inline _administrators/heroku.md]: Needs a period at the end. Missed this one before.

**Review by FrenjaminBanklin — APPROVED** [review]
One missing period I didn't notice before and one 'then' to 'than', otherwise looks fine to me.

**Pull request merged**